### PR TITLE
[FR] Change "le/de osu!" to "l/d'osu!" and "du osu!" to "de l'osu!"

### PR DESCRIPTION
**Pull request by Walavouchey**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review) @Biokax @DoryanR 

based on [discussion in discord](https://discord.com/channels/188630481301012481/218677502141399041/972559065249112184)

```sh
for file in $( find -type f -name "fr.md")
do
    perl -i -pe "s/\b([ldLD])e ((?:\**)|\[)os([ub])/\1'\2os\3/g" $file
    perl -i -pe "s/\b([dD])u ((?:\**)|\[)os([ub])/\1e l'\2os\3/g" $file
done
git restore wiki/People/The_Team/Staff_Log/2009/fr.md
```

### CODE REVIEW

**Review by DoryanR — APPROVED** [review]
Sounds good to me, thank you 👍

File: wiki/BanchoBot/fr.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ Il y a un total de 7 statuts qui peuvent être affichés par la commande `!stats
 
 ```
 13:04 pippi: !faq fr:peppy
-13:04 BanchoBot: Qui ne le connaît pas ? peppy est le créateur de osu!
```
Comment (TicClick) [inline wiki/BanchoBot/fr.md:138]: FWIW it's a direct quote from `!faq fr:peppy`, but I guess that could be fixed

**Review by TicClick — APPROVED** [review]

File: wiki/BanchoBot/fr.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/BanchoBot/fr.md:138]: very cool, how can i contribute to that

Comment (TicClick) [reply wiki/BanchoBot/fr.md:138]: I'll ask someone from the French GMs to proofread FAQs

Comment (Walavouchey) [reply wiki/BanchoBot/fr.md:138]: alright, tell them to also relay updates to this article or one of us

**Pull request merged**